app/assets/stylesheets/availabilities.scss

```diff
@@ -83,3 +133,196 @@
   top: -2px;
   float: left;
 }
+
+.availabilities-all-cell {
+  border: 1px solid #000 !important;
+}
+
+.availability-list {
+  list-style-type: none;
+  padding-left: 0;
+  margin-bottom: 0;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  white-space: nowrap;
+}
+
+li.text-overflow-ellipsis {
+  list-style-position: inside;
+}
+
+.body {
+  text-rendering: geometricPrecision;
+}
+
+html, body {
+  height: 100%;
+}
+
+.places-container {
+  height: calc(100% - 71px);
+  padding: 0!important;
+}
+
+.edit-places-form {
+  width: 100%;
+  height: 100%;
+  margin: 0!important;
+}
+
+.edit-places-nav {
+  height: 100%;
+  border-right: 1px solid $border-grey;
+  padding: 0!important;
+}
+
+.edit-places-table {
+  display: flex;
+  flex-direction: column;
+  flex: 20 20 auto;
+  height: 100%;
+}
+
+.edit-places-heading {
+  display: flex;
+  flex: none;
+}
+
+#scroll-table {
+  display: flex;
+  flex: 1 1 10%;
+  overflow-y: scroll;
+  border-top: 1px solid $border-grey;
+  align-items: flex-start;
+}
+
+.edit-places-empty-heading {
+  flex: 0 0 auto;
+  height: 100px;
+  width: 48px;
+  border-right: 1px solid $border-grey;
+}
+
+.heading-group {
+  flex: 1 1 auto;
+  display: flex;
+}
+
+.places-day-heading {
+  overflow: hidden;
+  display: flex;
+  align-items: flex-end;
+  flex: 1 1 0%;
+  border-right: 1px solid $border-grey;
+}
+
+.places-day-text {
+  margin-left: 10px;
+  font-size: 32px;
+  line-height: 64px;
+  overflow: hidden;
+}
+
+.time-heading {
+  flex: none;
+  width: 48px;
+  border-right: 1px solid $border-grey;
+  height: calc(24*80px);
+}
+
+.time-block {
+  height: 80px;
+  text-align: center;
+}
+
+.table-cells {
+  display: flex;
+  flex: 1 1 auto;
+  position: relative;
+}
+
+.table-columns {
+  flex: 1 1 0%;
+  border-right: 1px solid $border-grey;
+  padding-right: 12px;
+}
+
+.table-columns-padded {
+  width: 100%;
+  height: 100%;
+  position: relative;
+}
+
+.time-text {
+  position: relative;
+  top: -12px;
+  font-size: 10px;
+}
+
+::-webkit-scrollbar {
```

Comment:
You do not these vendor prefixes. The `autoprefixer` gem does this magically for you.